whitepaper/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+  <head>
+    <meta http-equiv="refresh" content="0; url=./whitepaper.pdf" />
+    <meta charset="UTF-8" />
+    <title>Wasm Interpreter Whitepaper</title>
+    <script type="text/javascript">
```

Comment:
`<meta http-equiv="refresh" content="0; url=http://example.com/" />` doesn't require JS and should work as well :)

Reply:
Though apparently one should in the best case combine the http-equiv, JS snippet and a user link.

Reply:
The file has a meta tag (see line 4)
`<meta http-equiv="refresh" content="0; url=./whitepaper.pdf" />`

Since this isn't blocking i'll just mark it as resolved and merge